en/debug/gdb_debugging.md

```diff
@@ -1,14 +1,8 @@
 # Embedded Debugging
```

Comment:
Should this file be renamed? Seems to be no longer about GDB.

Reply:
Yes, should be.